.github/workflows/playwright.yaml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
     playwright:
         name: "Playwright Tests"
-        runs-on: ubuntu-latest
+        runs-on: ubuntu-22.04
```

Comment:
It looks like Playwright isn't running out of the box on Ubuntu 24.04 at the moment due to some missing dependencies.

https://github.com/microsoft/playwright/issues/30368